go.mod

```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
 	cloud.google.com/go/pubsub v1.2.0
-	github.com/dangkaka/go-kafka-avro v0.0.0-20191011061810-51334c900b84
+	github.com/ericpubu/go-kafka-avro v0.0.2
```

Comment:
Is this fork not possible to do in the Merlin github organization?

Reply:
It's possible, but I'll keep upgrading in my fork to make a PR eventually on the main branch, so using that fork would give more functionalities

Reply:
as soon as the PR is merged I'll return the dependency to the main repo

Reply:
nice